src/common/utility_win.cpp

```diff
@@ -319,4 +319,31 @@ Utility::NtfsPermissionLookupRAII::~NtfsPermissionLookupRAII()
     qt_ntfs_permission_lookup--;
 }
 
+
+Utility::Handle::Handle(HANDLE h, std::function<void(HANDLE)> &&close)
+    : _handle(h)
```

Comment:
is the close function used anywhere so far? If not I'd rather not have it. You're only using the default, right?

Reply:
In the vfs plugin, the handle class is just moved to the main repo.